front/app/containers/Admin/reporting/containers/ReportBuilder/index.tsx

```diff
@@ -164,9 +164,13 @@ const ReportBuilder = ({
                       <ProjectTemplate
                         reportId={reportId}
                         projectId={templateProjectId}
+                        initializeContent={true}
```

Comment:
Not really used at the moment. But in case we make global reports public in the future.